_about_livecoms/03_earlycareer.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: single
+sidebar:
+  nav: earlycareer.md
+title: For Early Career Scientists 
+excerpt: How early career scientists can get involved with LiveCoMS
+permalink: /about/earlycareer/
+---
+
+We welcome early career scientists (undergraduate students, graduate
+students and postdocs) to help participate in LiveCoMS.  There are two
+ways you can participate:
+
+* If you are a graduate student or postdoc, we encourage you to sign
+  up to be a student reviewer. Please email
```

Comment:
I would insert a sentence here: "The *requirement* for a student reviewer for every submission is a unique strength of LiveCoMS that should enhance the pedagogical value of every accepted paper."

Reply:
Hmm. That doesn't really describe what it does for the _student_.  I adjusted a little bit here, and added more of this to the editor instructions.